client/python/README.md

```diff
@@ -0,0 +1,181 @@
+# Python client SDK for AGNTCY
```

Comment:
as this is directory API python SDK, we need to update the rest of the docs here to match that